docs/UseCases.md

```diff
@@ -67,8 +66,7 @@ These client libraries also require the context for the call they will join. Sim
 
 |Platform | Versions|
 |---------|---------|
-| iOS     | //NEED TO FILL IN |
-| Android | v23+    |
+| Android | v21+    |
```

Comment:
did we have problems with api version 21?

Reply:
We have two issues: I think, still API 21 can be supported with these issues. 
For API 21 and 22, calling SDK crashes on emulator
For API 21 and 22, UI library crashes if display name (remote participant) has emoji